src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/upload-part-icon.tpl.html

```diff
@@ -0,0 +1,73 @@
+                            <input ng-model="blade.imageUrl" type="url" placeholder="{{ 'demoSolutionFeaturesModule.blades.upload-part-icon.image-url.placeholder' | translate }}" tabindex="-1">
+                        </div>
+
+                        <div class="columns">
+                            <div class="column">
+                                <div class="form-group">
+                                    <label class="form-label">{{ 'demoSolutionFeaturesModule.blades.upload-part-icon.image-url.labels.download' | translate }}</label>
+                                    <div class="form-input">
+                                        <label class="form-label __switch">
+                                            <input type="checkbox" ng-model="blade.downloadImage">
+                                            <span class="switch"></span>
+                                        </label>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="column">
+                                <div class="form-group __right">
+                                    <a class="btn" ng-click="addImageFromUrl()">{{ 'demoSolutionFeaturesModule.blades.upload-part-icon.image-url.labels.add' | translate }}</a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+
+                    <div ng-show="uploader.isUploading">
+                        <div class="sub-t">{{ 'demoSolutionFeaturesModule.blades.upload-part-icon.labels.progress' | translate }}</div>
+                        <div class="progress-bar __aqua" ng-style="{'width': uploader.progress + '%'}"></div>
+                    </div>
+
+                    <div class="form-group">
+                        <div class="sub-t" ng-if-start="blade.currentEntity.imgSrc">{{ 'demoSolutionFeaturesModule.blades.upload-part-icon.preview.title' | translate }}</div>
+                        <table class="table preview" ng-if-end>
+                            <tbody>
+                                <tr class="table-item">
+                                    <td class="table-col __product-img">
+                                        <img ng-src="{{blade.currentEntity.imgSrc ? blade.currentEntity.imgSrc : ''}}">
+                                    </td>
+                                    <td class="table-col">
+                                        <div class="table-t">{{ blade.currentEntity.name ? blade.currentEntity.name : ('demoSolutionFeaturesModule.blades.upload-part-icon.preview.image.title' | translate) }}</div>
+                                        <div class="table-descr">
+                                            {{ 'demoSolutionFeaturesModule.blades.upload-part-icon.preview.image.description' | translate: { count: blade.currentEntity.partItems ? blade.currentEntity.partItems.length : 0 } }}
```

Comment:
Can we add a localization for single partItem case?

Reply:
Fixed, but in another way (too additional code : )

Reply:
I think we should use single and plural description. To reduce code base, we could use ternary operator.
Like in example: https://stackoverflow.com/questions/46350494/angular-translate-filter-in-ternary-operator